server.js

```diff
@@ -1,33 +1,140 @@
-import express from 'express'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
-
-// Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(express.json())
-
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+import express from "express";
+import cors from "cors";
+import listEndpoints from "express-list-endpoints"; // This is an express package to list all registered endoints and its verbs (npm i express-list-endpoints)
+import booksData from "./data/books.json";
+
+// Defines the port the app will run on. Defaults to 8080, but can be overridden when starting the server. For example: PORT=9000 npm start
+const port = process.env.PORT || 8080;
+const app = express(); // To initialised a new express server
+
+// Middlewares: to enable cors and json body parsing
+app.use(cors()); // Cors is a technology which allows APIs to say where request can come from for extra security
+app.use(express.json()); // express is exactly the same as the bodyParser() method
+
+
+
+//  --------- SIMPLE ENDPOINTS --------------//
+// Only use the response parameter as it gets a general response
+// Therefore, we don't need the request parameter to ask the user for a specific value.
+
+// Endpoint 1:  Front-end get just a sentence (http://localhost:8080)
+app.get("/", (req, res) => {
+  res.send(
+    "My name is Fatima and if you know me well, you must already know that I just love cats!"
+  );
```

Comment:
I think here, in your welcome message, you could write "add "/endpoints" in browser to view all endpoints", just to give the user a clearer idea how to use the API